receiver/kubeletstatsreceiver/documentation.md

```diff
@@ -8,27 +8,71 @@ These are the metrics available for this scraper.
+| **k8s.node.cpu.utilization** | Node CPU utilization | 1 | Gauge(Double) | <ul> </ul> |
+| **k8s.node.filesystem.available** | Node filesystem available | By | Gauge(Int) | <ul> </ul> |
+| **k8s.node.filesystem.capacity** | Node filesystem capacity | By | Gauge(Int) | <ul> </ul> |
+| **k8s.node.filesystem.usage** | Node filesystem usage | By | Gauge(Int) | <ul> </ul> |
+| **k8s.node.memory.available** | Node memory available | By | Gauge(Int) | <ul> </ul> |
+| **k8s.node.memory.major_page_faults** | Node memory major_page_faults | 1 | Gauge(Int) | <ul> </ul> |
+| **k8s.node.memory.page_faults** | Node memory page_faults | 1 | Gauge(Int) | <ul> </ul> |
+| **k8s.node.memory.rss** | Node memory rss | By | Gauge(Int) | <ul> </ul> |
+| **k8s.node.memory.usage** | Node memory usage | By | Gauge(Int) | <ul> </ul> |
+| **k8s.node.memory.working_set** | Node memory working_set | By | Gauge(Int) | <ul> </ul> |
+| **k8s.node.network.errors** | Node network errors | 1 | Sum(Int) | <ul> <li>interface</li> <li>direction</li> </ul> |
+| **k8s.node.network.io** | Node network IO | By | Sum(Int) | <ul> <li>interface</li> <li>direction</li> </ul> |
+| **k8s.pod.cpu.time** | Pod CPU time | s | Sum(Double) | <ul> </ul> |
+| **k8s.pod.cpu.utilization** | Pod CPU utilization | 1 | Gauge(Double) | <ul> </ul> |
+| **k8s.pod.filesystem.available** | Pod filesystem available | By | Gauge(Int) | <ul> </ul> |
+| **k8s.pod.filesystem.capacity** | Pod filesystem capacity | By | Gauge(Int) | <ul> </ul> |
+| **k8s.pod.filesystem.usage** | Pod filesystem usage | By | Gauge(Int) | <ul> </ul> |
+| **k8s.pod.memory.available** | Pod memory available | By | Gauge(Int) | <ul> </ul> |
+| **k8s.pod.memory.major_page_faults** | Pod memory major_page_faults | 1 | Gauge(Int) | <ul> </ul> |
+| **k8s.pod.memory.page_faults** | Pod memory page_faults | 1 | Gauge(Int) | <ul> </ul> |
+| **k8s.pod.memory.rss** | Pod memory rss | By | Gauge(Int) | <ul> </ul> |
+| **k8s.pod.memory.usage** | Pod memory usage | By | Gauge(Int) | <ul> </ul> |
+| **k8s.pod.memory.working_set** | Pod memory working_set | By | Gauge(Int) | <ul> </ul> |
+| **k8s.pod.network.errors** | Pod network errors | 1 | Sum(Int) | <ul> <li>interface</li> <li>direction</li> </ul> |
+| **k8s.pod.network.io** | Pod network IO | By | Sum(Int) | <ul> <li>interface</li> <li>direction</li> </ul> |
+| k8s.volume.available | The number of available bytes in the volume. | By | Gauge(Int) | <ul> </ul> |
+| k8s.volume.capacity | The total capacity in bytes of the volume. | By | Gauge(Int) | <ul> </ul> |
+| k8s.volume.inodes | The total inodes in the filesystem. | 1 | Gauge(Int) | <ul> </ul> |
+| k8s.volume.inodes.free | The free inodes in the filesystem. | 1 | Gauge(Int) | <ul> </ul> |
+| k8s.volume.inodes.used | The inodes used by the filesystem. This may not equal inodes - free because filesystem may share inodes with other filesystems. | 1 | Gauge(Int) | <ul> </ul> |
```

Comment:
Why these metrics are not enabled? I know this version of metrics builder doesn't support it, but we still should mark it correctly

Reply:
According to the [README](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/kubeletstatsreceiver#metric-groups) Volume metrics are not gathered by default.  [The existing code matches that statement](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/1c4dd67f7af084c467ce5378115fa151f7276547/receiver/kubeletstatsreceiver/config.go#L122).  I thought the `enabled` field in metadata.yaml is supposed to be `false` if the metric is not collected by default.

Reply:
You're right, but `MetricGroupsToCollect` is another setting. We need to deprecate it once we migrate to the new metrics builder and use `metrics -> enabled` instead. Until we do that, these metrics should be enabled, otherwise they wan't be emitted even if user adds Volume to the `MetricGroupsToCollect`.